### DISCUSSION TITLE
Strimzi KafkaMirrorMaker2 replication delay of 1 minute & no related consumer groups shown

### DISCUSSION THREAD

**Discussion by nithinelias** [strimzi/strimzi-kafka-operator#9411] [Q&A]
I have a strimzi kafka mirrormaker2 , single replica , deployed on an AKS cluster  . The mirrormaker connects to the source kafka cluster which is also a Strimzi kafka 3 node cluster in the same AKS and it's destination is an Azure Eventhub . I send a json message amounting to few bytes to the source topic and it takes a minute to see this in the eventhub . And this is quite consistent . What could be the reason ?My MM2 definition mostly resembles https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/mirror-maker/kafka-mirror-maker-2-tls.yaml except for the target cluster being eventhub in my case and my version is 3.5.1 . However when i test this locally on my laptop with strimzi kafka & mirrormaker deployed on docker desktop managed kubernetes , the messages are almost immediately received in the eventhub. How do i know when the mirrormaker read the message from source and when it sent to azure eventhub just from the MM2. The logs do not seem to show anything useful along these lines . Also i don't see any consumer groups corresponding to MM2 in the kafka cluster . One thing i noticed is that there some topic by the name mm2.internal** or so exist on my local kafka cluster while it doesn't exist on the kafka cluster deployed on AKS , could this have an effect ?

**Comment by scholzj** [comment]
It might be hard to help with this. Azure Event Hub is not really a Kafka cluster, so there might be many differences in how it behaves. If you see some missing topics when you compare it with running against Strimzi, it could definitely indicate some issues. You should properly check the logs if they don't have any errors about creating these topics etc.
